File: SpringBootAssignmentFrontEnd/online_book_store/src/app/model/feedback.ts
export class Feedback {
    feedbackId:number;
    feedbackRating:string;
    feedbackText:string;
    userId:number;
    userName:string;
    recommend:boolean;

    constructor(feedbackId:number,feedbackRating:string,feedbackText:string,userId:number,userName:string,recommend:boolean)
    {
        this.feedbackId=feedbackId;
        this.feedbackRating=feedbackRating;
        this.feedbackText=feedbackText;
        this.userId=userId;
        this.userName=userName;
        this.recommend=recommend;
    }
}

File: OnlineBookStore/src/main/java/com/cybage/model/Feedback.java
package com.cybage.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.ManyToOne;
import jakarta.persistence.Table;
import lombok.Data;

@Data
@Entity
@Table(name="feedback")
public class Feedback {
	
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private long feedbackId;
	private String feedbackRating;
	private String feedbackText;
	private String userName;
	private Boolean recommend;
	

	@ManyToOne
	private Users user;
	

}
